configs/published_toolchain_configs.json

```diff
@@ -1,2 +0,0 @@
-[]
```

Comment:
Can you specify somewhere (like a readme file) what the contents of this file are / will be and how is it generated

Reply:
This file has been deprecated and is being removed in this CL